sysl2/Lexer_test.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+    "testing"
+)
+
+func TestLexer(t *testing.T) {
+    content := `keywords:
+        while | return |
+        if | else | import`
+
+    regexes := make([]string, 7)
```

Comment:
Literal slice

Reply:
the indexes are used in writing the tokens array. I need it this way.